packages/ListBoxBrowser/src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import PropTypes from "prop-types";
+import RawButton from "@paprika/raw-button";
+import { button } from "./Breadcrumb.styles";
+import { isRoot } from "../../helpers";
+
+const propTypes = {
+  onClick: PropTypes.func.isRequired,
+  breadcrumb: PropTypes.arrayOf(PropTypes.object),
+};
+const defaultProps = {};
```

Comment:
`breadcrumb` prop is not required, do we need a default prop here? does eslint not complain?

Reply:
ah!!!! you right why is the linter not catching it. I will check this.